flink-quickstart/flink-quickstart-java/src/main/resources/archetype-resources/src/main/java/BatchJob.java

```diff
@@ -32,28 +32,29 @@
  * 		mvn clean package
  * in the projects root directory.
  * You will find the jar in
- * 		target/flink-quickstart-0.1-SNAPSHOT-Sample.jar
- *
+ * 		target/flink-quickstart-0.1-SNAPSHOT.jar
+ * From the cli you can then run
+ * 		./bin/flink run -c  &lt;BatchJob&gt; &lt;quickstart jar&gt;
  */
-public class Job {
+public class BatchJob {
 
 	public static void main(String[] args) throws Exception {
-		// set up the execution environment
+		// set up the batch execution environment
 		final ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
 
-
 		/**
 		 * Here, you can start creating your execution plan for Flink.
 		 *
 		 * Start with getting some data from the environment, like
-		 * 	env.readTextFile(textPath);
+		 * env.readTextFile(textPath);
```

Comment:
I think this space was intended formatting, like the below .filter(), .flatMap() etc.